services/task_manager.rb

```diff
@@ -0,0 +1,69 @@
+require_relative 'database_connection.rb'
+require_relative '../models/task.rb'
+
+class TaskManager
+  def change_the_status(task_id, new_status)
+    task = Task.find_task(task_id)
+    if task.nil?
+      puts "Status not changed due to invalid task id"
+    else
+      task.change_status(new_status)
+    end
+  end
+
+  def show_all_tasks
+    Task.show_all_tasks
+  end
+
+  def add_task(assignee_user_id:, description:, due_date:, priority:, user:)
+    user = User.find_by_email(user.email)
+    task = Task.create_task(assignee_user_id: assignee_user_id, description: description, due_date: due_date, priority: priority, user: user, status: 'pending') 
```

Comment:
split the line into multiple lines after it crosses 100 characters. Set up ruler on your text editor to indicate the same